# Request
float32 pos_x
float32 pos_y
float32 orientation_z
float32 orientation_w
---
# Response
string robot_name
string goal_location
---
# Feedback
float32 remained_dist